xarray/core/dataset.py

```diff
@@ -8424,6 +8709,40 @@ def argmin(self: T_Dataset, dim: Hashable | None = None, **kwargs) -> T_Dataset:
         -------
         result : Dataset
 
+        Example
+        -------
+
+        # Defined the dataset
+        >>> dataset = xr.Dataset(
+        ...     {
+        ...         "math_scores": (
+        ...             ["student", "test"],
+        ...             [[90, 85, 79], [78, 80, 85], [95, 92, 98]],
+        ...         ),
+        ...         "english_scores": (
+        ...             ["student", "test"],
+        ...             [[88, 80, 92], [75, 95, 79], [93, 96, 78]],
+        ...         ),
+        ...     },
+        ...     coords={
+        ...         "student": ["Alice", "Bob", "Charlie"],
+        ...         "test": ["Test 1", "Test 2", "Test 3"],
+        ...     },
+        ... )
+
+        # Indices of the minimum values along the 'student' dimension are calculated
+        >>> argmin_indices = dataset.argmin(dim="student")
```

Comment:
Is this argmin example duplicated from the previous PR?

Reply:
yes, both argmin and argmax are duplicated. Few other changes also are added, which I have not done. How do I delete these changes?

Reply:
You don't need to do the whole `git revert` thing in this case, it's fine to just delete those lines locally then do `git add` `git commit` as you would normally.